### PR TITLE
Update fr-FR.json

### PR DESCRIPTION
**Pull request by lyonelf**

### CODE REVIEW

File: mobile/assets/i18n/fr-FR.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
   "album_thumbnail_card_item": "1 élément",
   "album_thumbnail_card_items": "{} éléments",
   "album_thumbnail_card_shared": " · Partagé",
-  "album_thumbnail_owned": "Possédé",
+  "album_thumbnail_owned": "Propriétaire",
   "album_thumbnail_shared_by": "Partagé par {}",
-  "album_viewer_appbar_delete_confirm": "Are you sure you want to delete this album from your account?",
+  "album_viewer_appbar_delete_confirm": "Êtes-vous certains de vouloir effacer cet album de votre compte ?",
```
Comment (martabal) [inline mobile/assets/i18n/fr-FR.json:25]:
```suggestion
  "album_viewer_appbar_delete_confirm": "Êtes-vous certain de vouloir effacer cet album de votre compte ?",
```
Vous de politesse = `certain`
https://www.question-orthographe.fr/question/certain-ou-certains/

Comment (lyonelf) [reply mobile/assets/i18n/fr-FR.json]: Hello,

yes you are right. Except for Téléverser, it's really never used in common French speeking ( too long word)

"Charger" in french or "uploader" in frenglish

Comment (martabal) [reply mobile/assets/i18n/fr-FR.json]: IMO "charger" is ok but imprecise to describe an upload. What about "transférer" ?

Comment (lyonelf) [reply mobile/assets/i18n/fr-FR.json]: Maybe better yes

**Comment by alextran1502** [comment]
Stale, please send me your email so I can add you to the localization platform we use for these things

**Comment by lyonelf** [comment]
lyonelf@gmx.fr

**Pull request closed without merging**